src/etc/inc/ipsec.inc

```diff
@@ -1668,18 +1668,41 @@ function ipsec_setup_bypasslan() {
 	if (!empty($bypassnets)) {
 		$bypass = implode(',', $bypassnets);
 		$scconf['connections']['bypass'] = array();
-		/* Prevents the connection from being considered for remote peers */
-		$scconf['connections']['bypass']['remote_addrs'] = "127.0.0.1";
```

Comment:
This is absolutely required so that traffic matching the bypass TS does not ever try to initiate or be matched to a specific remote peer.

https://wiki.strongswan.org/projects/strongswan/wiki/UsableExamples#Passthrough-policy

Reply:
Ok, thanks for info

All fixed